html-forms_personal-profile-form/index.html

```diff
@@ -10,6 +10,28 @@
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
-    <section></section>
+    <section>
+     <form> <p>We want to get to know you! Please enter your personal information</p>
+      <fieldset>
+      <legend>Personal Information</legend>
+      <label for="firstName">First Name:</label>
+      <input name="firstName" id="firstName">
+      <label for="lastName">Last Name:</label>
+      <input name="lastName id="lastName">
+      <label name="birthDay" id="birthDay">Your Birthday:</label>
+      <input type="date" name="birthDay" id="birthDay"/>
+    </fieldset>
+    <p>Can you tell us a little bit more? We want to annoy you with unwated ads</p>
```

Comment:
same here I made a label, I don't know if its correct or not but that how I understood it

Reply:
Thanks. Will double-check and do the same if comfirms!